code/src/components/Sidebar.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export const Sidebar = (props) => {
+  return (
+    <>
+      <ul>
+        <li>
+          <a href={props.link}>{props.list}</a>
+          <p> Total tracks: {props.tracks}</p>
+        </li>
+      </ul >
+    </>
```

Comment:
If your component returns only one root element, there is no need to wrap it around React Fragment.